src/components/Login.jsx

```diff
@@ -16,10 +15,10 @@ const Login = ({ setUser }) => {
   const validateForm = () => {
     const newErrors = {};
 
-    if (!formData.username) {
-      newErrors.username = "Username is required";
-    } else if (formData.username.length < 3 || formData.username.length > 20) {
-      newErrors.username = "Username must be between 3 and 20 characters";
+    if (!formData.email) {
+      newErrors.email = "email is required";
+    } else if (formData.email.length < 3 || formData.email.length > 20) {
+      newErrors.email = "email must be between 3 and 20 characters";
     }
 
```

Comment:
```suggestion
}
```

I don't think theres any need to check email length, ideally all of our validation already does what is necessary